[Bug]: Merge CI, Simple · Diffusion & Model Executor Test, tests/model_executor/models/test_omni_processing.py, AttributeError: 'NoneType' object has no attribute 'mm_processor_cache_gb'

### Your current environment

<details>
<summary>The output of <code>python collect_env.py</code></summary>

```text
ci env
```

</details>


### Your code version

<details>
<summary>The commit id or version of vllm</summary>

```text
0.20.0
```
</details>
<details>
<summary>The commit id or version of vllm-omni</summary>

```text
dca369d448cd714d36bfaab7d54ab9e3449de306
```
</details>


### 🐛 Describe the bug

https://buildkite.com/vllm/vllm-omni/builds/9505/canvas?sid=019e1cd3-fa03-438d-a224-a27918aba08a&tab=output

```
[2026-05-12T15:46:10Z] =================================== FAILURES ===================================
--
  | [2026-05-12T15:46:10Z] _____ test_omni_processing_correctness[1.0-32-0.3-Qwen2_5OmniTalkerModel] ______
  | [2026-05-12T15:46:10Z]
  | [2026-05-12T15:46:10Z] model_arch = 'Qwen2_5OmniTalkerModel', hit_rate = 0.3, num_batches = 32
  | [2026-05-12T15:46:10Z] simplify_rate = 1.0
  | [2026-05-12T15:46:10Z]
  | [2026-05-12T15:46:10Z]     @pytest.mark.core_model
  | [2026-05-12T15:46:10Z]     @pytest.mark.omni
  | [2026-05-12T15:46:10Z]     @pytest.mark.cpu
  | [2026-05-12T15:46:10Z]     @pytest.mark.parametrize("model_arch", _get_model_archs_to_test())
  | [2026-05-12T15:46:10Z]     @pytest.mark.parametrize("hit_rate", [0.3, 0.5, 1.0])
  | [2026-05-12T15:46:10Z]     @pytest.mark.parametrize("num_batches", [32])
  | [2026-05-12T15:46:10Z]     @pytest.mark.parametrize("simplify_rate", [1.0])
  | [2026-05-12T15:46:10Z]     def test_omni_processing_correctness(
  | [2026-05-12T15:46:10Z]         model_arch: str,
  | [2026-05-12T15:46:10Z]         hit_rate: float,
  | [2026-05-12T15:46:10Z]         num_batches: int,
  | [2026-05-12T15:46:10Z]         simplify_rate: float,
  | [2026-05-12T15:46:10Z]     ):
  | [2026-05-12T15:46:10Z]         """
  | [2026-05-12T15:46:10Z]         For each Omni model verify that:
  | [2026-05-12T15:46:10Z]         - cached and non-cached processor outputs are identical.
  | [2026-05-12T15:46:10Z]         - text-prompt and token-prompt inputs yield the same result.
  | [2026-05-12T15:46:10Z]         """
  | [2026-05-12T15:46:10Z] >       _test_processing_correctness(
  | [2026-05-12T15:46:10Z]             model_arch,
  | [2026-05-12T15:46:10Z]             hit_rate=hit_rate,
  | [2026-05-12T15:46:10Z]             num_batches=num_batches,
  | [2026-05-12T15:46:10Z]             simplify_rate=simplify_rate,
  | [2026-05-12T15:46:10Z]         )
  | [2026-05-12T15:46:10Z]
  | [2026-05-12T15:46:10Z] tests/model_executor/models/test_omni_processing.py:328:
  | [2026-05-12T15:46:10Z] _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
  | [2026-05-12T15:46:10Z] tests/model_executor/models/test_omni_processing.py:173: in _test_processing_correctness
  | [2026-05-12T15:46:10Z]     model_config = _build_model_config(model_arch, info)
  | [2026-05-12T15:46:10Z]                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  | [2026-05-12T15:46:10Z] _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
  | [2026-05-12T15:46:10Z]
  | [2026-05-12T15:46:10Z] model_arch = 'Qwen2_5OmniTalkerModel'
  | [2026-05-12T15:46:10Z] info = _OmniExamplesInfo(default='Qwen/Qwen2.5-Omni-7B', model_stage='talker', hf_config_name='talker_config', has_multimodal...ers_version=None, max_transformers_version=None, is_available_online=True, trust_remote_code=False, max_model_len=None)
  | [2026-05-12T15:46:10Z]
  | [2026-05-12T15:46:10Z]     def _build_model_config(model_arch: str, info: _OmniExamplesInfo) -> OmniModelConfig:
  | [2026-05-12T15:46:10Z]         """Create an ``OmniModelConfig`` suitable for processor testing."""
  | [2026-05-12T15:46:10Z]         kwargs: dict = dict(
  | [2026-05-12T15:46:10Z]             model=info.default,
  | [2026-05-12T15:46:10Z]             tokenizer=info.default,
  | [2026-05-12T15:46:10Z]             tokenizer_mode="auto",
  | [2026-05-12T15:46:10Z]             trust_remote_code=info.trust_remote_code,
  | [2026-05-12T15:46:10Z]             model_arch=model_arch,
  | [2026-05-12T15:46:10Z]             model_stage=info.model_stage,
  | [2026-05-12T15:46:10Z]             max_model_len=info.max_model_len,
  | [2026-05-12T15:46:10Z]             enforce_eager=True,
  | [2026-05-12T15:46:10Z]             dtype="auto",
  | [2026-05-12T15:46:10Z]         )
  | [2026-05-12T15:46:10Z]         if info.hf_config_name is not None:
  | [2026-05-12T15:46:10Z]             kwargs["hf_config_name"] = info.hf_config_name
  | [2026-05-12T15:46:10Z]
  | [2026-05-12T15:46:10Z]         model_config = OmniModelConfig(**kwargs)
  | [2026-05-12T15:46:10Z]         # Ensure cache is effectively unlimited to prevent eviction during test
  | [2026-05-12T15:46:10Z] >       model_config.multimodal_config.mm_processor_cache_gb = 2048
  | [2026-05-12T15:46:10Z]         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  | [2026-05-12T15:46:10Z] E       AttributeError: 'NoneType' object has no attribute 'mm_processor_cache_gb'
  | [2026-05-12T15:46:10Z]
  | [2026-05-12T15:46:10Z] tests/model_executor/models/test_omni_processing.py:128: AttributeError
  | [2026-05-12T15:46:10Z] _____ test_omni_processing_correctness[1.0-32-0.5-Qwen2_5OmniTalkerModel] ______
  | [2026-05-12T15:46:10Z]
  | [2026-05-12T15:46:10Z] model_arch = 'Qwen2_5OmniTalkerModel', hit_rate = 0.5, num_batches = 32
  | [2026-05-12T15:46:10Z] simplify_rate = 1.0
  | [2026-05-12T15:46:10Z]
  | [2026-05-12T15:46:10Z]     @pytest.mark.core_model
  | [2026-05-12T15:46:10Z]     @pytest.mark.omni
  | [2026-05-12T15:46:10Z]     @pytest.mark.cpu
  | [2026-05-12T15:46:10Z]     @pytest.mark.parametrize("model_arch", _get_model_archs_to_test())
  | [2026-05-12T15:46:10Z]     @pytest.mark.parametrize("hit_rate", [0.3, 0.5, 1.0])
  | [2026-05-12T15:46:10Z]     @pytest.mark.parametrize("num_batches", [32])
  | [2026-05-12T15:46:10Z]     @pytest.mark.parametrize("simplify_rate", [1.0])
  | [2026-05-12T15:46:10Z]     def test_omni_processing_correctness(
  | [2026-05-12T15:46:10Z]         model_arch: str,
  | [2026-05-12T15:46:10Z]         hit_rate: float,
  | [2026-05-12T15:46:10Z]         num_batches: int,
  | [2026-05-12T15:46:10Z]         simplify_rate: float,
  | [2026-05-12T15:46:10Z]     ):
  | [2026-05-12T15:46:10Z]         """
  | [2026-05-12T15:46:10Z]         For each Omni model verify that:
  | [2026-05-12T15:46:10Z]         - cached and non-cached processor outputs are identical.
  | [2026-05-12T15:46:10Z]         - text-prompt and token-prompt inputs yield the same result.
  | [2026-05-12T15:46:10Z]         """
  | [2026-05-12T15:46:10Z] >       _test_processing_correctness(
  | [2026-05-12T15:46:10Z]             model_arch,
  | [2026-05-12T15:46:10Z]             hit_rate=hit_rate,
  | [2026-05-12T15:46:10Z]             num_batches=num_batches,
  | [2026-05-12T15:46:10Z]             simplify_rate=simplify_rate,
  | [2026-05-12T15:46:10Z]         )
  | [2026-05-12T15:46:10Z]
  | [2026-05-12T15:46:10Z] tests/model_executor/models/test_omni_processing.py:328:
  | [2026-05-12T15:46:10Z] _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
  | [2026-05-12T15:46:10Z] tests/model_executor/models/test_omni_processing.py:173: in _test_processing_correctness
  | [2026-05-12T15:46:10Z]     model_config = _build_model_config(model_arch, info)
  | [2026-05-12T15:46:10Z]                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  | [2026-05-12T15:46:10Z] _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
  | [2026-05-12T15:46:10Z]
  | [2026-05-12T15:46:10Z] model_arch = 'Qwen2_5OmniTalkerModel'
  | [2026-05-12T15:46:10Z] info = _OmniExamplesInfo(default='Qwen/Qwen2.5-Omni-7B', model_stage='talker', hf_config_name='talker_config', has_multimodal...ers_version=None, max_transformers_version=None, is_available_online=True, trust_remote_code=False, max_model_len=None)
  | [2026-05-12T15:46:10Z]
  | [2026-05-12T15:46:10Z]     def _build_model_config(model_arch: str, info: _OmniExamplesInfo) -> OmniModelConfig:
  | [2026-05-12T15:46:10Z]         """Create an ``OmniModelConfig`` suitable for processor testing."""
  | [2026-05-12T15:46:10Z]         kwargs: dict = dict(
  | [2026-05-12T15:46:10Z]             model=info.default,
  | [2026-05-12T15:46:10Z]             tokenizer=info.default,
  | [2026-05-12T15:46:10Z]             tokenizer_mode="auto",
  | [2026-05-12T15:46:10Z]             trust_remote_code=info.trust_remote_code,
  | [2026-05-12T15:46:10Z]             model_arch=model_arch,
  | [2026-05-12T15:46:10Z]             model_stage=info.model_stage,
  | [2026-05-12T15:46:10Z]             max_model_len=info.max_model_len,
  | [2026-05-12T15:46:10Z]             enforce_eager=True,
  | [2026-05-12T15:46:10Z]             dtype="auto",
  | [2026-05-12T15:46:10Z]         )
  | [2026-05-12T15:46:10Z]         if info.hf_config_name is not None:
  | [2026-05-12T15:46:10Z]             kwargs["hf_config_name"] = info.hf_config_name
  | [2026-05-12T15:46:10Z]
  | [2026-05-12T15:46:10Z]         model_config = OmniModelConfig(**kwargs)
  | [2026-05-12T15:46:10Z]         # Ensure cache is effectively unlimited to prevent eviction during test
  | [2026-05-12T15:46:10Z] >       model_config.multimodal_config.mm_processor_cache_gb = 2048
  | [2026-05-12T15:46:10Z]         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  | [2026-05-12T15:46:10Z] E       AttributeError: 'NoneType' object has no attribute 'mm_processor_cache_gb'
  | [2026-05-12T15:46:10Z]
  | [2026-05-12T15:46:10Z] tests/model_executor/models/test_omni_processing.py:128: AttributeError
  | [2026-05-12T15:46:10Z] _____ test_omni_processing_correctness[1.0-32-1.0-Qwen2_5OmniTalkerModel] ______
  | [2026-05-12T15:46:10Z]
  | [2026-05-12T15:46:10Z] model_arch = 'Qwen2_5OmniTalkerModel', hit_rate = 1.0, num_batches = 32
  | [2026-05-12T15:46:10Z] simplify_rate = 1.0
  | [2026-05-12T15:46:10Z]
  | [2026-05-12T15:46:10Z]     @pytest.mark.core_model
  | [2026-05-12T15:46:10Z]     @pytest.mark.omni
  | [2026-05-12T15:46:10Z]     @pytest.mark.cpu
  | [2026-05-12T15:46:10Z]     @pytest.mark.parametrize("model_arch", _get_model_archs_to_test())
  | [2026-05-12T15:46:10Z]     @pytest.mark.parametrize("hit_rate", [0.3, 0.5, 1.0])
  | [2026-05-12T15:46:10Z]     @pytest.mark.parametrize("num_batches", [32])
  | [2026-05-12T15:46:10Z]     @pytest.mark.parametrize("simplify_rate", [1.0])
  | [2026-05-12T15:46:10Z]     def test_omni_processing_correctness(
  | [2026-05-12T15:46:10Z]         model_arch: str,
  | [2026-05-12T15:46:10Z]         hit_rate: float,
  | [2026-05-12T15:46:10Z]         num_batches: int,
  | [2026-05-12T15:46:10Z]         simplify_rate: float,
  | [2026-05-12T15:46:10Z]     ):
  | [2026-05-12T15:46:10Z]         """
  | [2026-05-12T15:46:10Z]         For each Omni model verify that:
  | [2026-05-12T15:46:10Z]         - cached and non-cached processor outputs are identical.
  | [2026-05-12T15:46:10Z]         - text-prompt and token-prompt inputs yield the same result.
  | [2026-05-12T15:46:10Z]         """
  | [2026-05-12T15:46:10Z] >       _test_processing_correctness(
  | [2026-05-12T15:46:10Z]             model_arch,
  | [2026-05-12T15:46:10Z]             hit_rate=hit_rate,
  | [2026-05-12T15:46:10Z]             num_batches=num_batches,
  | [2026-05-12T15:46:10Z]             simplify_rate=simplify_rate,
  | [2026-05-12T15:46:10Z]         )
  | [2026-05-12T15:46:10Z]
  | [2026-05-12T15:46:10Z] tests/model_executor/models/test_omni_processing.py:328:
  | [2026-05-12T15:46:10Z] _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
  | [2026-05-12T15:46:10Z] tests/model_executor/models/test_omni_processing.py:173: in _test_processing_correctness
  | [2026-05-12T15:46:10Z]     model_config = _build_model_config(model_arch, info)
  | [2026-05-12T15:46:10Z]                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  | [2026-05-12T15:46:10Z] _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
  | [2026-05-12T15:46:10Z]
  | [2026-05-12T15:46:10Z] model_arch = 'Qwen2_5OmniTalkerModel'
  | [2026-05-12T15:46:10Z] info = _OmniExamplesInfo(default='Qwen/Qwen2.5-Omni-7B', model_stage='talker', hf_config_name='talker_config', has_multimodal...ers_version=None, max_transformers_version=None, is_available_online=True, trust_remote_code=False, max_model_len=None)
  | [2026-05-12T15:46:10Z]
  | [2026-05-12T15:46:10Z]     def _build_model_config(model_arch: str, info: _OmniExamplesInfo) -> OmniModelConfig:
  | [2026-05-12T15:46:10Z]         """Create an ``OmniModelConfig`` suitable for processor testing."""
  | [2026-05-12T15:46:10Z]         kwargs: dict = dict(
  | [2026-05-12T15:46:10Z]             model=info.default,
  | [2026-05-12T15:46:10Z]             tokenizer=info.default,
  | [2026-05-12T15:46:10Z]             tokenizer_mode="auto",
  | [2026-05-12T15:46:10Z]             trust_remote_code=info.trust_remote_code,
  | [2026-05-12T15:46:10Z]             model_arch=model_arch,
  | [2026-05-12T15:46:10Z]             model_stage=info.model_stage,
  | [2026-05-12T15:46:10Z]             max_model_len=info.max_model_len,
  | [2026-05-12T15:46:10Z]             enforce_eager=True,
  | [2026-05-12T15:46:10Z]             dtype="auto",
  | [2026-05-12T15:46:10Z]         )
  | [2026-05-12T15:46:10Z]         if info.hf_config_name is not None:
  | [2026-05-12T15:46:10Z]             kwargs["hf_config_name"] = info.hf_config_name
  | [2026-05-12T15:46:10Z]
  | [2026-05-12T15:46:10Z]         model_config = OmniModelConfig(**kwargs)
  | [2026-05-12T15:46:10Z]         # Ensure cache is effectively unlimited to prevent eviction during test
  | [2026-05-12T15:46:10Z] >       model_config.multimodal_config.mm_processor_cache_gb = 2048
  | [2026-05-12T15:46:10Z]         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  | [2026-05-12T15:46:10Z] E       AttributeError: 'NoneType' object has no attribute 'mm_processor_cache_gb'
  | [2026-05-12T15:46:10Z]
  | [2026-05-12T15:46:10Z] tests/model_executor/models/test_omni_processing.py:128: AttributeError
```


### Before submitting a new issue...

- [x] Make sure you already searched for relevant issues, and asked the chatbot living at the bottom right corner of the [documentation page](https://vllm-omni.readthedocs.io), which can answer lots of frequently asked questions.

--- a/tests/model_executor/models/registry.py
+++ b/tests/model_executor/models/registry.py
@@ -120,7 +120,6 @@ _OMNI_EXAMPLE_MODELS: dict[str, _OmniExamplesInfo] = {
         default="Qwen/Qwen2.5-Omni-7B",
         model_stage="talker",
         hf_config_name="talker_config",
-        has_multimodal_processor=True,
     ),
     "Qwen2_5OmniToken2WavModel": _OmniExamplesInfo(
         default="Qwen/Qwen2.5-Omni-7B",

